### ISSUE TITLE
Mining difficulty & data storing rising in price [critics]

### ISSUE THREAD

**Issue by ghost** [kevacoin-project/kevacoin#26]
I have no questions about the kevacoin cost by the market rates. But what I'm not understand - is the coins limitation and mining difficulty growing with the time. If we talking about kevacoin as the distributed database, what is the reason make this storage expansive in price with a time?

I think that current implementation will get the same issues like ethereum when storing the data there currently is not reasonable in the most of cases.

**Comment by kevacoin-project** [comment]
This is a valid concern but we do have a plan to deal with when it is too expensive to store data in the blockchain. The standard solution is sharding (the same scalability strategy ethereum will use to solve the scalability issue). Kevacoin is designed to be simple and efficient - much simpler than Ethereum and therefore it is much easier to support sharding.

**Comment by ghost** [comment]
> Kevacoin is designed to be simple and efficient - much simpler than Ethereum and therefore it is much easier to support sharding.

Agreed, as blogger, I love this project for that. Maybe the current (coin-based) model is a single good solution for the stable data storing reasons. Just thoughts, what is the future cost for a regular bloggers. But that will be another story, to care about that right now :)

Thank you for your work!

**Comment by d47081** [comment]
@kevacoin-project small block size limits does not resolve issue when user able to split same volume of data to smaller parts and send it anyway, cost is the same as one large or many of small particles, just last one make it less useful as the scalable DB

imo
